cve_bin_tool/checkers/perl.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+"""
+CVE checker for Perl
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-1885/Perl.html
+"""
+from ..util import regex_find
+
+
+def get_version(lines, filename):
+    """
+    returns version information for perl as found in a given file.
+    """
+    regex = [r"Perl: ([0-9]+\.[0-9]+\.[0-9]+)"]
```

Comment:
Be warned that this string doesn't actually work for perl 5.26 on Ubuntu, because the version string looks more like this:

```
This is perl 5, version 26, subversion 1 (%-p) built for x86_64-linux-gnu-thread-multi
```

So you'll need another regex in here for that.